pkgs/misc/emulators/ryujinx/updater.sh

```diff
@@ -1,10 +1,32 @@
-#!/usr/bin/env nix-shell
-#!nix-shell -i bash -p curl jq common-updater-scripts dotnet-sdk_5
+#! /usr/bin/env nix-shell
+#! nix-shell -i bash -p coreutils gnused curl common-updater-scripts nix-prefetch-git jq dotnet-sdk_5
 set -eo pipefail
 cd "$(dirname "${BASH_SOURCE[0]}")"
 
 deps_file="$(realpath "./deps.nix")"
 
+nix-prefetch-git https://github.com/ryujinx/ryujinx --quiet > repo_info
+new_hash="$(cat repo_info | jq -r ".sha256")"
+new_rev="$(cat repo_info | jq -r ".rev")"
+rm repo_info
+
+new_version="$(curl -s https://ci.appveyor.com/api/projects/gdkchan/ryujinx/branch/master | grep -Po '"version":.*?[^\\]",' | sed  's/"version":"\(.*\)",/\1/')"
```

Comment:
breaking this out into multiple lines would be nice

Reply:
How would you prefer to have this be broken up? I can't quite seem to format it in a nice looking way.

Reply:
does something like this work?

```suggestion
new_version="$(
    curl -s https://ci.appveyor.com/api/projects/gdkchan/ryujinx/branch/master \
        | grep -Po '"version":.*?[^\\]",' \
        | sed  's/"version":"\(.*\)",/\1/'
    )"
```